src/userConfig.js

```diff
@@ -0,0 +1,25 @@
+const keyMap = {
```

Comment:
Let's keep keyMap in its own file. Its application config. User can customize it, which we will implement later.